include/openthread/thread.h

```diff
@@ -423,7 +423,7 @@ OTAPI const otMeshLocalPrefix *OTCALL otThreadGetMeshLocalPrefix(otInstance *aIn
  * @param[in]  aMeshLocalPrefix  A pointer to the Mesh Local Prefix.
  *
  * @retval OT_ERROR_NONE           Successfully set the Mesh Local Prefix.
- * @retval OT_ERROR_INVALID_STATE  Thread protocols are enabled.
+ * @retval OT_ERROR_INVALID_STATE  Thread protocols are not enabled.
```

Comment:
Keep it as is?
Once thread protocols are enabled, Mesh Local Prefix should only be updated through dataset, but not allowed to change via API.